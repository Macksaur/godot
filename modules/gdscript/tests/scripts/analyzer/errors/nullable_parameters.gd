func test():
    var bar: String? = ""
    foo(bar, bar)


func foo(a: String?, b: String):
    prints(a, b)