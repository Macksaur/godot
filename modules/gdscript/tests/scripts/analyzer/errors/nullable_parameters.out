GDTEST_ANALYZER_ERROR
Invalid argument for "foo()" function: argument 2 must not be null, but "String?" is nullable.
